dashboard/src/api/resources/Resources.js

```diff
@@ -72,9 +68,7 @@ const local = false;
 
 export default class ResourcesAPI {
   static typeSenseClient = new TypesenseClient(
-    TYPESENSE_PORT,
-    TYPESENSE_URL,
-    TYPESENSE_API_KEY
+    API_URL + "/search/",
```

Comment:
The issue with this not working is that youre calling /search/ but theres no ingress route to direct that anywhere, so its defaulting to trying to load a page from the dashboard that doesnt exist

Reply:
added route